cypress/e2e/functional/tile_tests/question_tool_spec.js

```diff
@@ -0,0 +1,80 @@
+import Canvas from '../../../support/elements/common/Canvas';
+import ClueCanvas from '../../../support/elements/common/cCanvas';
+import QuestionToolTile from '../../../support/elements/tile/QuestionToolTile';
+
+const canvas = new Canvas;
+const clueCanvas = new ClueCanvas;
+const questionToolTile = new QuestionToolTile;
+let title = "QA 1.1 Solving a Mystery with Proportional Reasoning";
+let copyTitle = 'Question Tile Workspace Copy';
+
+function beforeTest() {
+  const queryParams = `${Cypress.config("qaUnitStudent5")}`;
+  cy.visit(queryParams);
+  cy.waitForLoad();
+}
+
+context('Question tool tile functionalities', function () {
+  it('add and delete question tile', function () {
+    beforeTest();
+
+    cy.log('adds question tile');
+    clueCanvas.addTile('question');
+    questionToolTile.getQuestionTile().should('exist');
+
+    // Question tile restore upon page reload
+    cy.wait(1000);
+    cy.reload();
+    cy.waitForLoad();
+    questionToolTile.getQuestionTile().should('exist');
+
+    cy.log('verifies restore of question tile');
```

Comment:
Should this log something like "verifies question tile does not automatically exist"?